lab_6_pipeline/pipeline.py

```diff
@@ -189,6 +301,13 @@ def _count_frequencies(self, article: Article) -> dict[str, int]:
         Returns:
             dict[str, int]: POS frequencies
         """
+        sentences_features = {}
+        str_conllu = str(self._analyzer.from_conllu(article).sentences)
```

Comment:
why do you cast it to str? it is very strange (!)

Reply:
changed it